src/java/fr/paris/lutece/util/http/SecurityUtil.java

```diff
@@ -54,13 +56,16 @@
     private static final String CONSTANT_HTTP_HEADER_X_FORWARDED_FOR = "X-Forwarded-For";
     private static final String PATTERN_IP_ADDRESS = "^([0-9]{1,3}\\.){3}[0-9]{1,3}$";
     private static final String CONSTANT_COMMA = ",";
+    private static final String CONSTANT_PATH_SEPARATOR = "." ;
     private static final String [ ] XXE_TERMS = {
             "!DOCTYPE", "!ELEMENT", "!ENTITY"
     };
     private static final String [ ] PATH_MANIPULATION = {
             "..", "/", "\\"
     };
 
+    public static final String PROPERTY_ANTPATHMATCHER_PATTERNS = "lutece.security.antpathmatcher.patterns";
+

```

Comment:
Maybe add a commented out entry in the default propertied file, with an explanation as to what this property does, when to use it, etc.